src/components/Button.tsx

```diff
@@ -1,12 +1,26 @@
-export default function Button({ buttonText, handleButtonClick, buttonWidth, buttonHeight }: { buttonText: string, handleButtonClick?: () => void, buttonWidth?: string, buttonHeight?: string }) {
-    const buttonStyle = `font-fira px-10 py-1 text-white bg-green rounded-md ${buttonWidth || 'w-52'} ${buttonHeight || 'h-7'}`;
-    return (
-        <button 
-            className={buttonStyle}
-            type="button"    
-            onClick={handleButtonClick}
-        >
-            {buttonText}
-        </button>
-    );
+export default function Button({
+  buttonText,
+  handleButtonClick,
+  buttonWidth,
+  buttonHeight,
+  buttonType,
+}: {
+  buttonText: string;
+  handleButtonClick?: () => void;
+  buttonWidth?: string;
+  buttonHeight?: string;
+  buttonType?: "button" | "submit" | "reset" | undefined;
+}) {
+  const buttonStyle = `font-fira px-10 py-1 text-white bg-green rounded-md ${
```

Comment:
It might be interesting for us to receive the buttonStyle as well as a parameter. There might be situations where we would want a larger button on certain screens, rounded or not, and other changing examples. What do you think of this change?

Reply:
Sounds great